static/src/js/reducers/query.js

```diff
@@ -4,10 +4,12 @@ const initialState = {
   collection: {
     pageNum: 1,
     spatial: {},
-    temporal: {}
+    temporal: {},
+    grid: ''
```

Comment:
I think we should change this to `gridName`, just to make it more clear what it is